.github/workflows/test.yml

```diff
@@ -34,7 +34,7 @@ jobs:
           xml: output.xml
           html: output.html
           markdown: output.md
-          subdomain-gateway-spec: false
+          specs: -subdomain-gateway
           args: -skip TestGatewayCar
```

Comment:
Disabled it by test name here. I think it's still useful in a case like this. Kubo does support CARs so it wouldn't make sense to disable testing CAR spec altogether. All we want is to disable this single test because Kubo's implementation is wrong at the moment.